src/app/routes/article/utils/mappings.ts

```diff
@@ -0,0 +1,47 @@
+const mappings = [
+  { name: 'recommendations', api: 'recommendations' },
+  {
+    name: 'datalabContentRecommendations',
+    engine: 'unirecs_datalab_content',
+    engineVariant: 'content',
+    api: 'datalab_content',
+  },
```

Comment:
Is this a new variation? Didn't see this one on the ticket. It could be a hold-over from the old experiment, I think the next experiment is fully `hybrid` recs. Worth checking with @JonBeeb?